docs/security.md

```diff
@@ -87,4 +90,4 @@ You can add or remove rules from this list as needed. After saving the rules to
 COMMIT
 ```
 
-Once the rules are loaded, install the `iptables-persistent` package in Ubuntu to re-apply them after every reboot.
\ No newline at end of file
+Once the rules are loaded, install the `iptables-persistent` package in Ubuntu to re-apply them after every reboot.
```

Comment:
I have no clue what's going on here, it looks the same. Maybe the online github text editor changed some invisible character, or the line ending?